task2/src/crudService/crudSrvice.ts

```diff
@@ -0,0 +1,66 @@
+type User = {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+  isDeselected: boolean;
+}
+
+const userData: User[] = [
```

Comment:
Static data should be moved to separate folder.

Reply:
done, ty